include/constants/pokemon.h

```diff
@@ -307,6 +307,7 @@
 #define EVO_SCRIPT_TRIGGER_DMG            36     // Pokémon has specified HP below max, then player interacts trigger
 #define EVO_DARK_SCROLL                   37     // interacts with Scroll of Darkness
 #define EVO_WATER_SCROLL                  38     // interacts with Scroll of Waters
+#define EVO_ITEM_USE_NIGHT                39     // specified item is used on Pokémon, is night
```

Comment:
```suggestion
#define EVO_ITEM_NIGHT                39     // specified item is used on Pokémon at night
```
The corresponding evolution method is called `EVO_ITEM` and phrased like that.